Jenkinsfile.github

```diff
@@ -97,6 +97,9 @@ pipeline {
                 stage('Init') {
                     steps{
                         script {
+                            // verify head of correlated to DOCS-CSM branch has been tagged (built)
+                            // need to explicitly provide name of correlated to DOCS-CSM branch
+                            sh "./verify-docs-csm-tag.sh main"
```

Comment:
will this go into other release branches? should this be `env.BRANCH_NAME` instead of `main`?

Reply:
I suppose I was thinking that this could always check main, and then if the branch is `release` or `stable` to just grab the number and then always verify against docs-csm:release/<num>.

Reply:
I tried using the values of env.BRANCH_NAME and env.GIT_BRANCH, but they both evaluate to the name of the current CSM tag, ie v1.3.0-beta.50. Further experimentation wasn't yielding better results, so I made the association explicit via the first parameter to verify-docs-csm-tag.sh.